content/en/opentelemetry/otlp_endpoint.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Datadog OTLP Trace Intake Endpoint
+private: true
+further_reading:
+  - link: "https://opentelemetry.io/docs/concepts/sdk-configuration/general-sdk-configuration/"
+    tag: "External"
+    text: "General OpenTelemetry SDK Configuration"
+  - link: "https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/"
+    tag: "External"
+    text: "OpenTelemetry Environment Variable Spec"
+  - link: "https://opentelemetry.io/docs/reference/specification/protocol/exporter/ "
+    tag: "External"
+    text: "OpenTelemetry Protocol Exporter"
+---
+{{< site-region region="ap1,gov" >}}
+<div class="alert alert-danger">This feature is not available for the {{< region-param key=dd_datacenter code="true" >}} site.</div>
```

Comment:
```suggestion
<div class="alert alert-warning">OTLP Trace Intake Endpoint is not supported for your selected  <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
```

This is how I've seen it done in other docs, so I'm putting it in line with those.